Adapter.C

```diff
@@ -757,6 +757,20 @@ void preciceAdapter::Adapter::storeMeshPoints()
 
 void preciceAdapter::Adapter::reloadMeshPoints()
 {
+    // Allow the user to say the mesh should not be moved, e.g. this is required
+    // for solids in solids4foam
+    if
+    (
+        !mesh_.lookupObject<IOdictionary>
```

Comment:
I agree it would be better to do this once during configuration. Let me know if I need to do something here, e.g. I can add the member variable and its initialisation if you prefer.

Reply:
Thanks.

I have checked and the solverType_ variable in FSI.H is "protected" so it is inaccessible from Adapter.C, e.g., adding the following line to "void preciceAdapter::Adapter::reloadMeshPoints()" in Adapter.C will not compile:
`FSI_->solverType_`.

What do you suggest? One option would be to add a public member function to FSI.H, e.g.
`    //- Return const-access to the solver type`
`    const std::string& solverType() const`
`    {`
`        return solverType_;`
`    }`
Or alternatively, to directly make the "solverType_" variable public.

Reply:
Hm, I see that this is a different case than the one we have inside each module. What if we just move this to the `setupCheckpointing` and store a more general boolean to the Adapter class?

Btw, does `mesh_.moving()` help here? We already use that in `storeMeshPoints()`.

Reply:
Ah, `mesh_.moving()` does help! Now we can avoid this ugly lookup. See abfe8c1.

This is much cleaner now.